lib/puppet/module_tool/skeleton/templates/generator/README.md.erb

```diff
@@ -13,67 +12,73 @@
 5. [Limitations - OS compatibility, etc.](#limitations)
 6. [Development - Guide for contributing to the module](#development)
```

Comment:
Reference and Limitations have become `###` below. Indent them here for consistency?